projects/CFT-Treasury/index.js

```diff
@@ -0,0 +1,15 @@
+const ADDRESSES = require('../helper/coreAssets.json')
+const { sumTokensExport, nullAddress, } = require('../helper/sumTokens')
+
+module.exports = {
+  tron: {
+    tvl: sumTokensExport({
+      tokensAndOwners: [
+        [nullAddress, 'TFvHNqDqttkXSS8ZTdC4c4W7q97SFW3iKq'],
```

Comment:
this is not a smart contract, is it your project treasury address?

Reply:
hmm, we cant just list with treasury part of the project, is the CFT staking part live now?

Reply:
CFT staking is live but the token is not listed for trading yet and therefor no value. So will add the staking part later. 

I was hoping to add the treasury that has 1,300,000 TRX staked. 

Here is the treasury address and I hope it can be added for now? 

https://tronscan.io/#/address/TFvHNqDqttkXSS8ZTdC4c4W7q97SFW3iKq

Reply:
hi, I would wait for CFT token to be tradable before listing, also, we dont traack treasury as tvl, but under treasury metric and keep the related adapters in `project/treasury/` folder